src/main/resources/assets/structurize/lang/en_us.json

```
@@ -1151,5 +1151,7 @@
   "block.structurize.multiblock": "Multi-Piston",
   "block.structurize.horizontal_plain_cactus_birch_timber_frame": "Plain Horizontal Cactus Birch Timber Frame",
   "structurize.gui.buildtool.creative_only": "Structurize does not support using the build tool when in survival. Switch to creative or install MineColonies and use the MineColonies Builder.",
-  "structurize.gui.replaceblock.ambiguous_properties": "Transformation from %s to %s is ambiguous because the following properties are not present in the source block: %s"
+  "structurize.gui.replaceblock.ambiguous_properties": "Transformation from %s to %s is ambiguous because the following properties are not present in the source block: %s",
+  "structurize.config.excludeentities": "Entities excluded from BP rendering",
```

What's BP? Just so every user knows explicitly.

---

I agree, blueprint should be spelled out

```suggestion
  "structurize.config.excludeentities": "Entities Excluded from Blueprint Rendering",
```

---

I agree with this but the name is just long, but could not find something longer
Feel free to suggest something completely different

---

It's in the config, doesn't matter if its a bit longer

---

config name is gonna be used for gui buttons, so they can't be long